code/modules/mob/living/carbon/carbon.dm

```diff
@@ -215,8 +210,8 @@
 			AdjustKnockdown(-3)
 			playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 			M.visible_message( \
-				"<span class='notice'>[M] shakes [src] trying to wake [t_him] up!</span>", \
-				"<span class='notice'>You shake [src] trying to wake [t_him] up!</span>", \
+				"<span class='notice'>[M] shakes [src] trying to wake \him up!</span>", \
```

Comment:
Because the rest of it is at least relevant to reference fixing. This has nothing to do with anything, and I'm not having this file change 2x (twice) in the merge + revert for no good reason.